go.mod

```diff
@@ -3,14 +3,37 @@ module github.com/edgexfoundry/device-sdk-go/v2
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
 	github.com/OneOfOne/xxhash v1.2.8
-	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.1.0
+	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da
+	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/davecgh/go-spew v1.1.1
+	github.com/eclipse/paho.mqtt.golang v1.3.5
+	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.1.1-dev.2
+	github.com/edgexfoundry/go-mod-configuration/v2 v2.1.0
 	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.1.0
-	github.com/edgexfoundry/go-mod-messaging/v2 v2.1.0
+	github.com/edgexfoundry/go-mod-messaging/v2 v2.1.1-dev.2
 	github.com/edgexfoundry/go-mod-registry/v2 v2.1.0
+	github.com/edgexfoundry/go-mod-secrets/v2 v2.1.1-dev.1
+	github.com/fxamacker/cbor/v2 v2.3.0
+	github.com/go-kit/kit v0.11.0
+	github.com/go-logfmt/logfmt v0.5.0
+	github.com/go-playground/validator/v10 v10.9.0
+	github.com/go-redis/redis/v7 v7.3.0
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
+	github.com/hashicorp/consul/api v1.9.1
+	github.com/hashicorp/go-immutable-radix v1.0.0
+	github.com/hashicorp/go-multierror v1.1.0
+	github.com/mitchellh/consulstructure v0.0.0-20190329231841-56fdc4d2da54
+	github.com/mitchellh/copystructure v1.0.0
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/pebbe/zmq4 v1.2.7
 	github.com/pelletier/go-toml v1.9.4
+	github.com/pmezard/go-difflib v1.0.0
+	github.com/stretchr/objx v0.1.1
 	github.com/stretchr/testify v1.7.0
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4
+	golang.org/x/sys v0.0.0-20210806184541-e5e7981a1069
+	golang.org/x/text v0.3.6
```

Comment:
Should all of these non edgex-go-mods be changing?

Reply:
Humm, looks like they got added

Reply:
Going to recreate the PR